### DISCUSSION TITLE
Uninstall Symfony

### DISCUSSION THREAD

**Discussion by tokyoabc** [symfony/symfony#51072] [Q&A]
I want to completely uninstall Symfony on my machine but I don't know how, can anyone help me, I use Windows 11 operating system.

**Comment by lyrixx** [comment]
How did you install it?

**Comment by epitre** [comment]
@tokyoabc If you used scoop, `scoop uninstall symfony-cli` should do it.  
https://github.com/ScoopInstaller/Scoop/wiki/Commands
